src/components/common/HistoryPopup.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import "./styles.css";
+import ReactModal from "react-modal-resizable-draggable";
```

Comment:
Any reason we couldn't use the ant-design model here? https://ant.design/components/modal/
Looks like you added a brand new dependency